packages/style-engine/prepare-unit-tests-for-backporting.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I reckon we don't really need something like this. I'll remove it in a future commit.

Doing it via IDE is just as fast and means less work.

I just wanted to make sure we don't need to worry that the PHP unit tests no longer live in the package file. 

The Core tests exist as well so I think we're covered.

Reply:
I kinda love the idea of including bash scripts somewhere to make this process a little easier. But yes, maybe in another PR? There could potentially be a good home for it in [`/bin/`](https://github.com/WordPress/gutenberg/tree/trunk/bin) somewhere?